documentation/docs/pipelines/cloud-sdk/configuration.md

```diff
@@ -523,126 +508,8 @@ Also, the stage (and thus an extension) is only executed if a stage configuratio
 
 ## Step configuration
 
-### artifactPrepareVersion
-
-The pipeline can be configured to store release candidates in a Nexus repository after they passed all stages successfully.
-By default, the pipeline will perform automatic versioning of artifacts via the step `artifactPrepareVersion`.
-This ensures that multiple builds of a continuously delivered application do not lead to version collisions in Nexus.
-If you are not building a continuously delivered application, you will typically disable automatic versioning.
-To do this, set the value of the parameter `versioningType` to the value `library`.
-Architectural details can be found in [automatic-release.md](https://github.com/SAP/cloud-s4-sdk-pipeline/blob/master/doc/architecture/decisions/automatic-release.md).
-
-Example:
-
-```yaml
-steps:
-  artifactPrepareVersion:
-    versioningType: library
-```
-
-For more information on how to configure this step, please refer to the documentation in [project "Piper"](https://sap.github.io/jenkins-library/steps/artifactPrepareVersion/).
-
-### mavenExecute
-
-The mavenExecute step is used for all invocations of the mvn build tool. It is either used directly for executing specific maven phases such as `test`, or indirectly for steps that execute maven plugins such as `checkPmd`.
-
-| Property | Mandatory | Default Value | Description |
-| --- | --- | --- | --- |
-| `dockerImage` | | `maven:3.6.1-jdk-8-alpine` | The image to be used for executing maven commands. Please note that at least maven 3.6.0 is required. |
-| `projectSettingsFile` | | | The project settings.xml to be used for maven builds. You can specify a relative path to your project root or a URL starting with http or https. |
-
-### mavenExecuteStaticCodeChecks
-
-The mavenExecuteStaticCodeChecks step executes static code checks for maven based projects. The tools SpotBugs and PMD are used. For more information on how to configure this step please refer to the documentation in [project "Piper"](https://sap.github.io/jenkins-library/steps/mavenExecuteStaticCodeChecks/).
-
-### executeNpm
```

Comment:
Not sure what we should do about this one. Technically it is still in our library, and in some places it is still used, if I am not mistaken.